.github/workflows/release-helm-charts.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: release-helm-charts
+
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+
+jobs:
+  trigger-release:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.action == 'published' && !startsWith(github.ref, '8.16') && !startsWith(github.ref, '8.17') }}
```

Comment:
I mixed two different things:

```
      - run: echo 'hi ${{ github.ref_name }}'
      - run: echo 'hi ${{ github.ref }}'
```

given 8.16.2, it produced

<img width="333" alt="image" src="https://github.com/user-attachments/assets/80cceed4-89a9-48ec-ac6b-5a35e336eadc" />


```suggestion
    if: ${{ github.event.action == 'published' && !startsWith(github.ref_name, '8.16') && !startsWith(github.ref_name, '8.17') }}
```